docs/data/overview.md

```diff
@@ -2,7 +2,11 @@
 layout: docu
 title: Importing Data
 ---
-The first step to using a database system is to insert data into that system. DuckDB provides several data ingestion methods that allow you to easily and efficiently fill up the database. In this section, we provide an overview of these methods so you can select which one is correct for you.
+<!-- not sure if this section should be abstracted into its own page, or where it would be best placed, feel free to move -->
+### Create a Connection (in-memory or persistant file)
```

Comment:
@Mytherin that makes sense. These updates were a result of me searching the documentation for confirmation on how to create persistent db files, & not finding anything. Maybe a "Connect & persist a database" section would make sense?

Reply:
A `Connect` section would be fine, although the problem with that is that this is different for every single client. It could contain links to the pages of different clients, perhaps?

Reply:
@Mytherin yea, I included updates in the `Client APIs` section for some specific clients where I could infer the proper approach (R, Python, Rust). Those are also part of this PR. 

But wasn't sure where to put this generic instruction (which also refers & links to the `API docs` section for client-specific usage). And wasn't sure if it was worth its own whole page. There also seems to be some redundancy between the `Guides` & `Documentation` sections, so wasn't sure where is best: 
<img width="200" alt="Screen Shot 2022-03-29 at 4 21 31 PM" src="https://user-images.githubusercontent.com/18430230/160700381-e961a5e8-b680-4a27-9c1b-627e8510881e.png">

As a new user, I was simply looking for info on creating & persisting database files, & could find no mentions in the docs. So my suggestion again would be something like an explicit `Connect & persist a database` section in either `Guides` or `Documentation`, but will defer to the team.

Reply:
> But wasn't sure where to put this generic instruction (which also refers & links to the API docs section for client-specific usage). And wasn't sure if it was worth its own whole page. There also seems to be some redundancy between the Guides & Documentation sections, so wasn't sure where is best:

The Guides are meant as quick guides to accomplish specific goals, whereas the documentation (which should perhaps be renamed to `Reference`) is meant as a comprehensive reference that describes all available options. The guides serve as a starting point that point to the documentation section.

> As a new user, I was simply looking for info on creating & persisting database files, & could find no mentions in the docs. So my suggestion again would be something like an explicit Connect & persist a database section in either Guides or Documentation, but will defer to the team.

The problem is that there is no generic way of connecting to and persisting a database. How to do this depends on which client you are using. Perhaps for now it is better to just leave this bit out, and leave the information only in the specific client API sections.

Reply:
Sounds good, perhaps only include the link once:

> ### Connect or Create a DB 
> To use DuckDB, you must first create a connection to a database. The exact process varies by client. Most clients take a parameter pointing to a database file to read and write from (the file extension may be anything, e.g. `.db`, `.duckdb`, etc.). If the database file does not exist, it will be created. The special value `:memory:` can be used to create an **in-memory database** where no data is persisted to disk (i.e. all data is lost when you exit the process). See the [API docs](docs/api/overview) for client-specific details. 

Could you then add a `Connect` page at the top of the `Documentation` that includes this?

Reply:
I've got that change ready to commit! 

Do I need to do anything other than add a `connect.md` file to the `docs/` folder? (& undo the change to `overview.md`)

e.g. any changes to `_data/menu_docs_*` to add the page to the sidebar menu?

Reply:
Yes, you need to change the menu_docs as well when creating a new page.

Reply:
I have committed the changes: 

- reverted `overview.md` changes
- added [connect.md](https://github.com/duckdb/duckdb-web/pull/193/files#diff-8e766a984a7cccac610c71411a140ac72108b17e849077b58f4d068afb2da13f)  
- updated [menu_docs_current.json](https://github.com/duckdb/duckdb-web/pull/193/files#diff-f61b38e2f9a878690155277cc3e45bd42f789a3d76b0a1719fbaa9d86a397120) (not 100% confident I did this right, so worth a check)